qemu/qemu.go

```diff
@@ -2440,6 +2440,9 @@ type Config struct {
 	// CPUModel is the CPU model to be used by qemu.
 	CPUModel string
 
+	// SandboxOn is the qemu function which enables the seccomp feature
+	SandboxOn string
```

Comment:
@LiangZhou-CTY Can you name this config option accordingly to reflect it is related to seccomp, something like `SeccompSandbox` ?

Reply:
Will qemu have to be built with seccomp for this, not sure if is enabled in our current config options :https://github.com/kata-containers/kata-containers/tree/2fc7f75724ac9e18e60f63dcc9aa395dc51c184d/tools/packaging/qemu/default-configs
 cc @devimc